### PR TITLE
feat: added pt-BR localization

### PR DESCRIPTION
**Pull request by darlantc**
## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [x] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

Added Brazilian Portuguese localization file

### CODE REVIEW

**Review by SokratisVidros — APPROVED** [review]
👏

**Comment by SokratisVidros** [comment]
@darlantc Thanks a lot for the contribution!

File: packages/localizations/src/pt-BR.ts [hunk 1]
```diff
@@ -0,0 +1,546 @@
...
+      title: 'Verifique seu celular',
+      subtitle: 'para continuar em {{applicationName}}',
+      formTitle: 'Código de verificação',
+      formSubtitle: 'Insira o código enviado para seu celular',
+      resendButton: 'Reenviar código',
+    },
+  },
+} as const;
+
+export const enUS: DeepRequired<LocalizationResource> = {
```
Comment (SokratisVidros) [inline packages/localizations/src/pt-BR.ts:15]:
```suggestion
export const ptBR: DeepRequired<LocalizationResource> = {
```

**Review by SokratisVidros — CHANGES_REQUESTED** [review]

File: packages/localizations/src/pt-BR.ts (hunk 1 above)
Comment (nikosdouvlis) [reply packages/localizations/src/pt-BR.ts:15]: We also need to export this from `./index.ts`. Happy to do it myself but I don't want to hijack your PR :)

Thanks for the contribution!

**Review by SokratisVidros — APPROVED** [review]

**Comment by nikosdouvlis** [comment]
Thank you @darlantc !

**Pull request merged**